src/watch.js

```diff
@@ -0,0 +1,92 @@
+import { configRead, configAddChangeListener } from './config';
+import './watch.css';
+import { requireElement } from './screensaver-fix.ts';
+
+class Watch {
+  #watch;
+  #timer;
+  #attrChanges;
+  #PLAYER_SELECTOR = 'ytlr-watch-default';
+
+  constructor() {
+    this.createElement();
+    this.startClock();
+    this.playerEvents();
+  }
+
+  createElement() {
+    this.#watch = document.createElement('div');
+    this.#watch.className = 'webOs-watch';
+    document.body.appendChild(this.#watch);
+  }
+
+  startClock() {
+    const nextSeg = (60 - new Date().getSeconds()) * 1000;
+
+    const formatter = new Intl.DateTimeFormat(navigator.language, {
+      hour: 'numeric',
+      minute: 'numeric'
+    });
+
+    const setTime = () => {
+      this.#watch.innerText = formatter.format(new Date());
+    };
+
+    setTime();
+    setTimeout(() => {
+      setTime();
+      this.#timer = setInterval(setTime, 60000);
+    }, nextSeg);
+  }
+
+  playerAppear(video) {
+    this.changeVisibility(video);
+    this.playerObserver(video);
+  }
+
+  changeVisibility(video) {
+    const focused = video.getAttribute('hybridnavfocusable') === 'true';
+    this.#watch.style.display = focused ? 'none' : 'block';
+  }
+
+  async playerEvents() {
+    const player = await requireElement(this.#PLAYER_SELECTOR, HTMLElement);
+    this.playerAppear(player);
+  }
+
+  playerObserver(node) {
+    this.#attrChanges = new MutationObserver((mutations) => {
+      for (const mutation of mutations) {
```

Comment:
This `for` loop is unnecessary at best since you base the CSS `display` value on a conditional instead of manually tracking.